scripts/download_from_gdrive.py

```diff
@@ -22,7 +22,7 @@ def parse(self, response):
             response.xpath('//div[@class="uc-main"]'
                            + '/div[@id="uc-text"]/a/@href').extract()[0]
         yield Request(url=downlaod_url, callback=self.save_file,
-                      meta={'download_maxsize' : 0})
+                      meta={'download_maxsize' : 0, 'download_timeout' : 600})
```

Comment:
The download is the slowest part. Depending on the internet connectivity it may be well above 10min. Maybe make this slightly higher? Do we even need a timeout here?